README.md

```diff
@@ -1 +1,187 @@
 # Work In Progress - Onfido Java Library
+
+The official Java library for integrating with the Onfido API.
+
+Documentation can be found at <https://documentation.onfido.com>
+
+This library is only for use on the backend, as it uses Onfido API tokens which must be kept secret. If you do need to collect applicant data in the frontend of your application, we recommend that you use one of [the Onfido SDKs](https://developers.onfido.com/sdks/).
+
+## Installation
+
+_Coming soon_
+
+## Getting Started
+
+Import the `Onfido` object, this is the main object used for interfacing with the API:
+
+```java
+import com.onfido.Onfido;
+```
+
+Instantiate and configure an `Onfido` instance with your API token, and region if necessary:
+
+```java
+Onfido onfido = Onfido.builder()
+                .apiToken(<YOUR_API_TOKEN_HERE>)
+                .regionUS() // Defaults to EU region, add this to switch to US
+                .build();
+```
+
+## Making a call to the API
+
+Using your configured and instantiated instance of the `Onfido` object you can make calls to the API by calling one of the methods on the resources inside of it. For example:
+
+```java
+onfido.applicant.create(<REQUEST_BODY_HERE>);
+```
+
+All request bodies can be created using a builder pattern on the `Request` subclass of the desired resource. For example in the request above an `Applicant.Request` object is needed as the body, this would look something like:
+
+```java
+Applicant.request().firstName("First").lastName("Last");
+```
+
+The above will return an Applicant.Request object that can be provided to the `create()` method as the request body, so a full call to to the API will look something like:
+
+```java
+Applicant newApplicant = onfido.applicant.create(
+    Applicant.request().firstName("First").lastName("Last")
+);
+```
+
+The above method will return the corresponding resource object, as most methods will. In this particular case an `Applicant` object will return with all information present for the newly created applicant. To access the information call the getter of the desired property on the object, for example:
+
+```java
+newApplicant.getFirstName();
+```
+
+## Resources
+
+All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.list` to fetch all resources.
+
+#### Applicants
+
+Applicants are the object upon which Onfido checks are performed.
+
+```java
+onfido.applicant.create(requestBody)                    # => Creates an applicant
```

Comment:
Minor: Could we use the same convention for request bodies as in the above example, i.e. `<REQUEST_BODY_HERE>`

Same for other sections